sklearn/tree/tests/test_tree.py

```diff
@@ -2392,8 +2392,8 @@ def test_min_sample_split_1_error(Tree):
 
 
 @pytest.mark.parametrize("criterion", ["squared_error", "friedman_mse"])
-def test_missing_values_on_equal_nodes_no_missing(criterion):
-    """Check missing values goes to correct node during predictions"""
+def test_missing_values_best_splitter_dtc_on_equal_nodes_no_missing(criterion):
+    """Check missing values goes to correct node during predictions for DTC."""
```

Comment:
I don't think we should specify dtc because it seems like a short form for DecisionTreeClassifier whereas the test uses DecisionTreeRegressor
```suggestion
def test_missing_values_best_splitter_on_equal_nodes_no_missing(criterion):
    """Check missing values goes to correct node during predictions."""
```